modules/rhel-adding-node.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-adding-node-{context}"]
+= Adding a RHEL compute machine to your cluster
+
+You can add compute machines that use Red Hat Enterprise Linux as the operating
+system to a {product-title} {product-version} cluster.
+
+.Prerequisites
+
+* You installed the required packages and performed the necessary configuration
+on the machine that you run the playbook on.
+* You prepared the RHEL hosts for installation.
+
+.Procedure
+
+Perform the following steps on the machine that you prepared to run the
+playbook:
+
+. Extract the pull secret for your cluster.
++
+----
+$ oc -n openshift-config get -o jsonpath='{.data.\.dockerconfigjson}' secret pull-secret | base64 -d | jq .
+----
+
+. Save the pull secret in a file that is named `pull-secret.txt`.
+
+. Create an Ansible inventory file that is named `/<path>/inventory/hosts` that
+defines your compute machine hosts and required variables.
++
+----
+[all:vars]
+ansible_user=root <1>
+#ansible_become=True <2>
+
+openshift_kubeconfig_path="~/.kube/config" <3>
```

Comment:
absolute path is recommended.
or probably meet error 
"stderr": "error: stat ~/.kube/config: no such file or directory"
at TASK [openshift_node : Get release image]
as normal user sudo when execute this task
no problem when absolute path used 
openshift_kubeconfig_path="/home/wmeng/.kube/config"

https://github.com/openshift/openshift-ansible/blob/release-4.1/roles/openshift_node/tasks/config.yml#L51
  delegate_to: localhost

Reply:
@mtnbikenc has a fix out for it and has confirmed that the string in the doc is correct.